### DISCUSSION TITLE
add quemu-guest-agent for use on Proxmox

### DISCUSSION THREAD

**Discussion by HonkXL** [mikopbx/Core#719] [General ( Обо всём )]
Hi,
when using MikoPBX on Proxmox, everything works fine. But the qemu-guest-agent is not installed on the Linux and I found no way to do so. On Debian it's apt-get install qemu-guest-agent
Do you think it would be possible to add the qemu-guest-agent to the image or make a script or anything else that makes a installation possible?

**Comment by boffart** [comment]
There are no such plans. This package pulls too many dependencies.

**Comment by jorikfon** [comment]
Часто спрашивают, можно попробовать в внешний модуль вынести такое, чтобы в базу не тянуть.
Есть еще проекты этого тулза на python
https://github.com/xolox/python-negotiator
